MangaJaNaiConverterGui/backend/src/pyproject.toml

```diff
@@ -13,8 +13,8 @@ dependencies = [
   "sanic==24.6.0",
   "spandrel_extra_arches==0.1.1",
   "spandrel==0.3.4",
-  "torch==2.1.2",
-  "torchvision==0.16.2"
+  "torch==2.3.0",
+  "torchvision==0.18.0"
```

Comment:
This is the only potentially breaking change in this MR. I'm not sure if it's okay to introduce it, but my attempts at running it on old version failed miserably. It was complaining at other sub-dependencies.

Reply:
Thanks for the PR! I actually observed some performance regressions with torch versions after 2.1.2 (on Windows at least) which is why I was using that version. I noticed that chaiNNer's integrated Python also uses torch 2.1.2, does chaiNNer's integrated Python work for you? If it does, I wonder what the difference is here.

Reply:
I'll double check in a free moment and let you know

Reply:
I think I found the culprit. On clean venv with only these deps installed I was getting a runtime error saying that numpy cannot be initialized because some package was built with 1.x and I'm using 2.0.1. It happened somewhere in torch code.

I checked the dependency tree and it seems that torchvision requires "any" version. So pip is installing 2.0.1 which is the newest one ¯\\_(ツ)_/¯

It's working now, but I am wondering - could you please check what numpy version you are using when running the code?